### PR TITLE
added footer to all pages, finalized capri page layout

### PR DESCRIPTION
**Pull request by atphy**

### CODE REVIEW

File: capris.html [hunk 1]
```diff
@@ -6,8 +6,10 @@
   <title>Document</title>
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
   <link rel="stylesheet" href="main.css">
+  <script src="https://kit.fontawesome.com/3c70a135dc.js" crossorigin="anonymous"></script>
```
Comment (ryanbeiden) [inline capris.html]: Should this be at the bottom of the ```body``` tag?

File: contact.html [hunk 2]
```diff
@@ -6,8 +6,10 @@
   <title>Document</title>
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
   <link rel="stylesheet" href="main.css">
+  <script src="https://kit.fontawesome.com/3c70a135dc.js" crossorigin="anonymous"></script>
```
Comment (ryanbeiden) [inline contact.html]: Same here

File: history.html [hunk 3]
```diff
@@ -6,8 +6,10 @@
   <title>Document</title>
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
   <link rel="stylesheet" href="main.css">
+  <script src="https://kit.fontawesome.com/3c70a135dc.js" crossorigin="anonymous"></script>
```
Comment (ryanbeiden) [inline history.html]: And here lol

File: index.html [hunk 4]
```diff
@@ -10,6 +10,7 @@
     <script src="https://kit.fontawesome.com/3c70a135dc.js" crossorigin="anonymous"></script>
```
Comment (ryanbeiden) [inline index.html]: I guess here too which was there in previous commits

**Review by ryanbeiden — CHANGES_REQUESTED** [review]
Not sure if the ```script``` tag in the head of each ```html``` file needs to be moved to the bottom of the ```body```

File: capris.html (hunk 1 above)
Comment (atphy) [reply capris.html]: moved to bottom of body tag

**Review by ryanbeiden — APPROVED** [review]
💪

**Pull request merged**